package.json

```diff
@@ -49,10 +49,12 @@
     "chai-as-promised": "^7.1.1",
     "decentraland-commons": "^4.0.1",
     "decentraland-eth": "^8.8.0",
+    "dotenv": "^6.0.0",
```

Comment:
I deleted this and my node_modules, reinstalled and everything works! Is it really necessary? It can conflict with decentraland-common's dotenv